docs/docs/firmware/can-traffic.md

```diff
@@ -1,38 +1,46 @@
----
-title: Calculating CAN traffic on a bus
----
 
+# Calculating CAN traffic
 
+In order to calculate the CAN load, we must declare a few variables and their importance within the CAN protocol.
 
-In order to calculate the CAN load, we must declare a few variables and their importance within the CANbus protocol. 
-
-1. Baud: (kbaud)
+## Baud (kbaud)
 
 The baud rate is the rate at which data is transfered in a communication channel (like CAN). For example, 9600 Baud rate is 9600 bits per second
 
-2. Frequency (Hz)
+## Frequency (Hz)
 
-Frequency in the CANbus refers to how many CAN frames (or messages) are transmitted per second. So the greater the frequency, the more bits transferred, increasing the bus load. 
+Frequency is the number of messages transmitted per second for each message type. Different messages can have different frequencies.
```

Comment:
Can you reword this whole line like "The frequency of a message, in `Hz` is how many times it is transmitted per second."